ginga/rv/plugins/LineProfile.py

```diff
@@ -155,29 +180,39 @@ def build_gui(self, container):
         container.add_widget(top, stretch=1)
         self.gui_up = True
 
+        self.select_mark(self._new_mark)
         self.build_axes()
 
     def build_axes(self):
         if (not self.gui_up) or (self.hbox_axes is None):
             return
         self.hbox_axes.remove_all()
-        self.selected_axis = None
         self.clear_plot()
 
         image = self.fitsimage.get_image()
         if image is not None:
             # Add Checkbox widgets
             # `image.naxispath` returns only mdim axes
-            for i in range(1, len(image.naxispath)+3):
-                chkbox = Widgets.CheckBox('NAXIS%d' % i)
+            nx = len(image.naxispath)
+            maxi = nx + 2
+            for i in range(1, maxi + 1):
+                chkbox = Widgets.CheckBox('NAXIS{}'.format(i))
                 self.hbox_axes.add_widget(chkbox)
 
                 # Disable axes for 2D images
-                if len(image.naxispath) <= 0:
+                if nx <= 0:
+                    self.selected_axis = None
                     chkbox.set_enabled(False)
-                else:
-                    # Add callback
-                    self.axes_callback_handler(chkbox, i)
+                    continue
+
+                # Add callback
+                self.axes_callback_handler(chkbox, i)
+
```

Comment:
I attempted to set default NAXIS by doing `chkbox.set_state(True)` around here. But it results in Ginga crashing with core dump. Not sure where else I can set it.